src/hooks-worker-client.coffee

```diff
@@ -301,7 +301,8 @@ class HooksWorkerClient
 
     @runner.hooks.afterAll (transactions, hookCallback) =>
 
-      # This is needed for transaction modification integration tests.
+      # This is needed for transaction modification integration tests:
+      # https://github.com/apiaryio/dredd-hooks-template/blob/master/features/execution_order.feature
```

Comment:
This won't easily move as all the hooks handler implementations use the `dredd-hooks-template` repo as a base, so we don't change it much (actually, I'd even say we don't change it at all). I'd use commit if there would be a link to specific line, but this way I'd say it's as stable as any other internet link.